doc/LocalVMSetup.md

```diff
@@ -16,8 +16,18 @@ You can download CentOS 7.8.2003 ISO file from http://www.centos.org, or from an
 ## 3. Install a fresh CentOS 7.8.2003
 Automatic install or manual install. Basic installation is OK. When you build CORTX from source in next steps, dependant packages will be resolved and installed.
 
-## 4. You may need to add EPEL repo.
-Please refer to: https://fedoraproject.org/wiki/EPEL
+## 4. Run the following commands.
+
+$ sudo yum install epel-release
+
+$ sudo yum repolist
+
+$ sudo yum install ansible
+
+$ wget https://archives.fedoraproject.org/pub/archive/epel/5/x86_64/epel-release-5-4.noarch.rpm
```

Comment:
Shouldn't it be `epel-release-latest-7` for CentOS-7 as mentioned in the link which you removed?

Reply:
That didn't work for me when I used that command